x-pack/legacy/plugins/reporting/export_types/png/server/create_job/index.ts

```diff
@@ -4,18 +4,12 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import {
-  ServerFacade,
-  RequestFacade,
-  ConditionalHeaders,
-  CreateJobFactory,
-} from '../../../../types';
+import { ServerFacade, RequestFacade, ConditionalHeaders } from '../../../../types';
 import { validateUrls } from '../../../../common/validate_urls';
 import { cryptoFactory } from '../../../../server/lib/crypto';
-import { oncePerServer } from '../../../../server/lib/once_per_server';
 import { JobParamsPNG, ESQueueCreateJobFnPNG } from '../../types';
 
-function createJobFn(server: ServerFacade) {
+export function createJobFn(server: ServerFacade): ESQueueCreateJobFnPNG {
```

Comment:
https://github.com/elastic/kibana/pull/49426/files#diff-4dd055be05123b17234a25322487fc0dR12

Reply:
Oops, bug: should be `createJobFactory`